### DISCUSSION TITLE
SSL Manager extension download for Docker Image

### DISCUSSION THREAD

**Discussion by phackwer** [nextgenhealthcare/connect#4695] [Q&A]
Hi,

I've contacted sales and got a development license key. I've added the environment variable LICENSE_KEY as I can see it's read on entrypoint.sh and I also found the EXTENSIONS_DOWNLOAD documented in the readme for the docker on docker hub. My problem is that I can't find where this extension can be downloaded from. Also, it's not clear enough if I need to add all extensions on a single zip or not. 

Does anybody knows where the extensions can be downloaded from?

Thanks.

**Comment by pacmano1** [comment]
Extensions are downloaded at the customer portal. You need a login and password for that. I would reach back out to the sales guy to get the information.￼

**Comment by phackwer** [comment]
Thanks. I've emailed them earlier, but the timezone difference means I will wait a while now. :-D 

Freetime = write documentation time! yeeeee.. :+1:

**Reply by ralphb918** [reply]
Were you ever able to set up your SSL Manager? If not we have an option for you as well at Zen Healthcare IT. Thx
